chainer/functions/parameter.py

```diff
@@ -22,6 +22,13 @@ def __init__(self, array):
         self.W = array
         self.gW = numpy.empty_like(array)
 
+    def __call__(self, volatile=False):
+        ret = super(Parameter, self).__call__()
```

Comment:
I think the value of `rank` can be arbitrary even if `volatile` is `False`
Since `__call__` method does not take input variables, `Parameter` function is a leaf node of computational graph (DAG). Traverse backward of computational graph does not impose anything on the `rank` of root variables.